sanic/config.py

```diff
@@ -1,8 +1,19 @@
-import os
-import types
+from os import environ as os_environ
+
+from typing import Union, \
+                   Any
+
+from .utils import str_to_bool, \
+                   load_module_from_file_location
+
+
+
+# NOTE (tomaszdrozdz): remove in version: 21.3
+from deprecated import  from_envvar, \
+                        from_pyfile, \
+                        from_object
+# END remove in version: 21.3
```

Comment:
About this deprecated methods - how about that aproach ?

Reply:
Looks good, but I'm not sure that we need `# END ` comment in the end.

Also, would be great to provide a bit more details in the `NOTE` section, it will be helpful for other contributors, why it will be removed/reasons etc.

Reply:
OK.